code/index.html

```diff
@@ -12,28 +12,39 @@
   </head>
   <body>
     <aside class="question-section">
-      <button type="button" id="restart" class="outlined-button">RESTART</button>
+      <button type="button" id="restart" class="outlined-button">
+        RESTART
+      </button>
 
       <img
         class="guess-who-icon"
         src="images/guess-who-bubble.png"
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Does the person have...</h1>
 
       <select title="questions" id="questions">
         <optgroup label="hair">
```

Comment:
Maybe you can "finlira" on the hair color suggestions. I saw that there is big letters, small letters and a ?

:)

Reply:
🙏🏻👊🏻👍🏼